### ISSUE TITLE
Screen reader does not read combo-box list items

### ISSUE THREAD

**Issue by vsccarl** [Microsoft/vscode#6492]
- VSCode Version: 1.1.0-alpha
- OS Version: Windows 10

Steps to Reproduce:

Using NVDA
1. Open the output box
2. Click on the combo box to change between Extensions/Git/Tasks

Also try using the combo box for selecting a debug configuration. It just reads "combo box expanded, List" and will not read any of the listed items.

**Comment by isidorn** [comment]
I nicely hear the options in the combo box If I use OS X and voice over and use keyboard to go over the options.
Thus it is a bug in nvda, or how chrome handles the html select element on windows - nothing we can do here it is native behavior.
